src/AdminApp/ManageSalons.js

```diff
@@ -0,0 +1,249 @@
+        const response = await fetch(
+          `http://localhost:8080/bounds-services/${category}`,
+          {
+            method: "POST",
+            body: JSON.stringify({
+              serviceName: category,
+              LatBtm: bounds._ne.lat,
+              LonBtm: bounds._sw.lng,
+              LatTop: bounds._sw.lat,
+              LonTop: bounds._ne.lng,
+              distance: 0,
+              treatment: null,
+              priceFrom: 0,
+              priceTo: 0,
+              opinion: 0.0,
+              localizationLat: latitude,
+              localizationLon: longitude,
+            }),
+            headers: {
+              "Content-Type": "application/json",
+            },
+          }
+        );
+        let json = [];
+
+        if (response.ok) {
+          json = await response.json();
+          setSalons(json);
+        } else {
+          alert("Nie znaleziono rekordow  w bazie danych dla " + category);
+        }
+
+        console.log(json);
+        console.log("Salons i category i bounds");
+      }
+    }
+    load();
+  }, [salons]);
+
+  useEffect(() => {
+    async function load() {
+      if (category !== null && bbox !== null && salons.length === 0) {
+        const response = await fetch(
+          `http://localhost:8080/services/${category}`,
+          {
+            method: "POST",
+            body: JSON.stringify({
+              serviceName: category,
+              LatBtm: bbox.btmRightPoint?.lat,
+              LonBtm: bbox.btmRightPoint?.lon,
+              LatTop: bbox.topLeftPoint?.lat,
+              LonTop: bbox.topLeftPoint?.lon,
+              distance: 0,
+              treatment: null,
+              priceFrom: 0,
+              priceTo: 0,
+              opinion: 0.0,
+              localizationLat: latitude,
+              localizationLon: longitude,
+            }),
+            headers: {
+              "Content-Type": "application/json",
+            },
+          }
+        );
+        let json = [];
+
+        if (response.ok) {
+          json = await response.json();
+          setSalons(json);
+        } else {
+          alert("Nie znaleziono rekordow  w bazie danych dla " + category);
+        }
+
+        console.log(salons.length);
+        console.log("Salons i category i bbox");
+      }
+    }
+    load();
+  }, [salons]);
+
+  useEffect(() => {
+    function success(position) {
+      setLatitude(position.coords.latitude);
+      setLongitude(position.coords.longitude);
+    }
+
+    function errorF() {
+      setError("Unable to retrieve your location");
+    }
+
+    navigator.geolocation.getCurrentPosition(success, errorF);
+  }, []);
+
+  return (
+    <>
+      <div className="App" style={{ height: "100vh" }}>
+        <div className="MapContainer">
+          <Menu setCategory={setCategory} />
+          <Map
+            mapStyle="https://api.tomtom.com/map/1/style/20.0.1-1/basic_main-lite.json?key=1ncwaIygtJ0KrjH5ssohlEKUGFf7G5Dv"
```

Comment:
`key` also can be defined in the `constants` file (if it isn't your personal key, if it's your private key it shouldn't be committed to git). If key changed you don't have to search for it in all files and replace in all places that it is used - you can do it once.